apps/web/src/components/UpcomingLessons/Content.tsx

```diff
@@ -134,6 +159,10 @@ export const Content: React.FC<{
                   image: otherMember.image,
                   role: otherMember.role === "student" ? "student" : "tutor",
                 }}
+                sendingMessage={sendingMessage === item.lesson.id}
+                sendMsgBtnDisabled={
+                  sendingMessage !== 0 && sendingMessage !== item.lesson.id
+                }
```

Comment:
* Just `disabled`, we can use it a prop to disable the main button of the card. 
* All buttons including the "sending button" should be disabled.

```suggestion
                disabled={!!sendingMessage}
```